defaults/main.yml

```diff
@@ -1,6 +1,9 @@
 ---
 # defaults file for ansible-tdagent
 
+# td-agent version e.g 2 ,3 ..
+td_agent_version: 3
```

Comment:
does it require handling "major / minor" version number.
If yes, rename to "td_agent_repository_major_version" or use ansible filter if available to extract the major version part.

Reply:
this only controls the major version as the os by default would install and update the latest minor version. so If you want I can rename it to "td_agent_repository_major_version"